config/mfDefs.gem

```diff
@@ -34,11 +34,14 @@ UserStaticLinkFlags  =
 UserDynamicLinkFlags =
 
 IncludeDirs+=$(XDAQ_ROOT)/include
+IncludeDirs+=/usr/local/include/python2.7
 LibraryDirs+=$(XDAQ_ROOT)/lib
 DependentLibraryDirs+=$(XDAQ_ROOT)/lib
+DependentLibraryDirs+=/usr/local/lib
 
 DependentLibraries+=asyncresolv boost_system config log4cplus
 DependentLibraries+=numa toolbox uuid xcept xdaq2rc xerces-c
+DependentLibraries+=python2.7
 
```

Comment:
No, apparently there’s a dependency in the gemsupervisor I believe, that’s why I added it there. 
I've checked the full compilation, it fails if the changes are made only to the gemutils/Makefile 
> On Jul 1, 2016, at 3:22 PM, Jared Sturdy <notifications@github.com> wrote:
> 
> In config/mfDefs.gem <https://github.com/jsturdy/cmsgemos/pull/7#discussion_r69298344>:
> 
> >  
> >  DependentLibraries+=asyncresolv boost_system config log4cplus
> >  DependentLibraries+=numa toolbox uuid xcept xdaq2rc xerces-c
> > +DependentLibraries+=python2.7
> >  
> These changes should go in the gemutils/Makefile since not all packages need to be linked against python
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/jsturdy/cmsgemos/pull/7/files/3b27a9d2dfd8b60ed208160c82333abe5c65613d..dc9ad51b01bef9050314442c58e04891a6f00200#r69298344>, or mute the thread <https://github.com/notifications/unsubscribe/AHFalrGOFtnb2q1JPbm6bENYRzS6ksjpks5qRRSwgaJpZM4JCH8N>.
>